scripts/set_gem_timing

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+NC_STRING="NC"
+SC_STRING="SC"
+# TODO:(josh) reflect on absolute paths
+IOC_PATH_NC="/reg/g/pcds/epics/ioc/fee/GasDetDAQ/R4.0.22"
+IOC_PATH_SC="/reg/g/pcds/epics/ioc/fee/GasDetDAQ/R4.0.22-NOTS"
```

Comment:
I think canonically in bash scripts you'd use environment variables to generalize something like this (maybe someone has a better suggestion?), but I don't think you need to worry about it here. Given the expected lifetime of the mode of operations, the only risk I see is if we need to update the ioc versions and then we'd need to come back and update this script again (and also make sure people stop using the old version of the script...)

Reply:
Great point, I think an analog of this comment is in my TODOs section. I would like to understand how we do it here I just didn't know how much of the path should be environment-variable-ized. 

Agree with lifetime point; hopefully the TODO is reminder enough for us to address. If you would prefer tracking outside of source code let me know.

Reply:
It's in the source code as a comment and there's a record of discussions here and in the PR text, so that should be enough.

Reply:
Even better news. @mcb64 let me know `imgr` infers release area, we do not need full path